Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/PressableButtons/PressableButton.cs

```diff
@@ -130,6 +130,11 @@ private void Start()
             }
         }
 
+        private void OnEnable()
```

Comment:
I believe @thalbern actually fixes this in a very similar way in #4150 so I don't think it's needed any more, but thank you for suggesting it.